packages/forms/src/widgets/EnumerationWidget/EnumerationWidget.js

```diff
@@ -77,6 +77,10 @@ class EnumerationWidget extends React.Component {
 		};
 	}
 
+	componentWillReceiveProps(nextProps) {
+		this.setState({ ...this.state, items: nextProps.formData });
```

Comment:
in the constructor the formData object seems to be reworked : 
`items: (props.formData || []).map((item) => ({
				values: item.values,
			})),`

Just wondering why there's a difference in the willReceiveProps ? In that case, the items property directly being assigned the formData object in props.